parts/ltsp/client/lib/has-real-keyboard

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -eu
+set -o pipefail
+
+while read -a handlers; do
+    for handler in "${handlers[@]}"; do
+        if [ "${handler}" = sysrq ]; then
```

Comment:
tested on a couple of devices:
linux smartphone (no keyboard): ok
surface without keyboard: ok
hp laptop: ok
intel nucs, however, seem to have a
`N: Name="ACPI Virtual Keyboard Device"`
always present, which has a handler list of
`H: Handlers=sysrq kbd event4 rfkill`
(checked remotely a couple of devices; didn't physically verify that they don't have a keyboard connected right now, but at least from one I'm pretty confident it does not, and another one had a separate "AT Translated Set 2 keyboard" which is probably what a real connected keyboard looks like. but I can verify that later this week)